sale_wishlist/models/res_partner.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class ResPartner(models.Model):
+
+    _inherit = 'res.partner'
+
+    wishlists_count = fields.Integer(
+        compute='_compute_wishlists_count',
+        string="# Wishlists"
+    )
+
+    def _compute_wishlists_count(self):
+        # do just one query for all records
+        data = self.env['product.set'].read_group([
+            ('partner_id', 'in', self.ids)
```

Comment:
```suggestion
            ('partner_id', 'in', self.ids), ('typology', '=', 'wishlist')
```
@simahawk The domain to compute the count is not the same as the one given to action IOW, if a product set exists for a given partner the smart button will display '1 wishlist but once we'll click on the action, the list will be empty since the action's domain include a filter on the typology.